containerd/containerd.go

```diff
@@ -120,8 +122,24 @@ func (d *Driver) loadContainer(id string) (containerd.Container, error) {
 	return d.client.LoadContainer(d.ctxContainerd, id)
 }
 
-func (d *Driver) createTask(container containerd.Container) (containerd.Task, error) {
-	return container.NewTask(d.ctxContainerd, cio.NewCreator(cio.WithStdio))
+func (d *Driver) createTask(container containerd.Container, stdoutPath, stderrPath string) (containerd.Task, error) {
+	stdout, err := openFIFO(stdoutPath)
```

Comment:
Is there an equivalent call for Windows?

Reply:
This is just opening a file (`os.OpenFile()`) on the filesystem (With the only difference being, the file in question is a named pipe).
`openFIFO` is defined on `L141`